### DISCUSSION TITLE
Using recipes with Blitz v0.27.1

### DISCUSSION THREAD

**Discussion by PaulH-Dev** [blitz-js/blitz#1617] [General] [answered]
Very early days, but I'm loving Blitz so far - thank you!

Recipes look very handy, I'm hoping they can reduce dependency pain, but I've not managed to get them to work for me.

I've tried `blitz install reflexjs` and accepted all the defaults.

This made a number of changes to code, including adding this line to `_app.tsx`:
```
import { ThemeProvider } from "reflexjs";
```

reflexjs isn't yet installed, so I tried:
```
$ npm install reflexjs --save
```
which resulted in the following output:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: blitzstack@1.0.0
npm ERR! Found: react@0.0.0-experimental-4ead6b530
npm ERR! node_modules/react
npm ERR!   react@"0.0.0-experimental-4ead6b530" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.14.0" from reflexjs@1.0.1
npm ERR! node_modules/reflexjs
npm ERR!   reflexjs@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
```

Similar dependency issues when I backed out and tried `blitz install chakra`.

Anyone know where I'm going wrong?

**Comment by flybayer** [comment, marked as answer]
Hey @PaulH-Dev! Looks like the issue is that you are using npm v7 which breaks most published npm packages 😞 

Try running `npm install --legacy-peer-deps` to fix it.

**Reply by PaulH-Dev** [reply]
Hi @flybayer, I was indeed running v7, and that option fixed my problem.

Thanks so much!
